test-utils/tests/tracing_macros.rs

```diff
@@ -0,0 +1,74 @@
+use std::{
+    thread::sleep,
+    time::{Duration, Instant},
+};
+
+use libcrux_test_utils::tracing::{EventType, MutexTrace, Trace as _, TraceEvent};
+
+mod str_labels {
+
+    use super::*;
+
+    lazy_static::lazy_static! {
+        static ref TRACE_STR: MutexTrace<&'static str, Instant> = Default::default();
+    }
```

Comment:
lazy static isn't needed anymore for things like this. You could use `LazyLock` for example.